internal/store/postgres/migrations/000016_replace_idempotencies_success_notification_id_column.down.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE
+  notifications
+DROP COLUMN IF EXISTS receiver_selectors;
+
```

Comment:
@mabdh why we are dropping -> adding and again dropping the receiver_selectors column.

Reply:
It is up and down. Am I missing something there?

ref
https://github.com/golang-migrate/migrate

Reply:
@manishdangi98 there are the up and down queries. The up file is used to implement the changes and down file is used for rollback. Down is the reverse if Up.